src/harmony/Alert/Alert.tsx

```diff
@@ -28,7 +28,7 @@ export const Alert: React.FC<AlertProps> = ({ text, icon, className, view, ...pr
     return (
         <div className={cn(s.Alert, viewMap[view], className)} {...props}>
             {nullable(icon, (i) => i, iconMap[view])}
-            <Text color="inherit">{text}</Text>
+            <Text>{text}</Text>
```

Comment:
Does we really need Text component here now?

Reply:
no